client/src/components/Game.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Player from './Player';
+
+import AddPlayer from './AddPlayer';
+
+class Game extends Component {
```

Comment:
as of now this component doesn't hold state so I would make this a pure component

```javascript
import React from 'react';
import Player from './Player';

import AddPlayer from './AddPlayer';

export default (props) => {
  const { players, deck } = props.game;
  return(
    <div>
      <AddPlayer addPlayer={props.addPlayer} />
      {(players.length > 1) ? <button onClick={props.startGame.bind(null, players, deck)}>Start Game</button> : ''}
      {players.map((p, i) => <Player key={i} player={p}/>)}
    </div>
  )
}
```